src/main/java/org/jabref/gui/externalfiles/FindUnlinkedFilesViewModel.java

```diff
@@ -0,0 +1,123 @@
+package org.jabref.gui.externalfiles;
+
+import java.io.FileFilter;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.scene.control.TreeItem;
+import javafx.stage.FileChooser;
+
+import org.jabref.gui.util.FileFilterConverter;
+import org.jabref.logic.util.StandardFileType;
+
+import com.sun.star.util.DateTime;
```

Comment:
Please use the java 8 date classes, the LocalDateTime:  https://www.baeldung.com/java-8-date-time-intro